app/src/main/java/com/khmelenko/lab/varis/log/LogEntryComponent.java

```diff
@@ -0,0 +1,7 @@
+package com.khmelenko.lab.varis.log;
+
+public abstract class LogEntryComponent {
```

Comment:
what was the reason to have this as an abstract class and not interface?

Reply:
No special reason behind it. I can change it to an interface as well.